workflow/kfdrc_RNAseq_workflow.cwl

```diff
@@ -65,8 +65,8 @@ doc: >-
   
   The related Github branch for this app is located [here](https://github.com/kids-first/kf-rnaseq-workflow/tree/be-publish).
     
-id: kfdrc-rnaseq-wf
-label: Kids First DRC RNA-Seq Workflow
+id: kfdrc-rnaseq-single-fq-modified-wf
+label: Kids First DRC RNA-Seq Single FASTQ Modified Workflow
```

Comment:
Is this just to make it stand out from the existing workflow? It can still handle both single and paired reads correct?

Reply:
Correct, it was just so that the name was not the same in Cavatica